features/zoek-met-burgerservicenummer/gba/autorisatie-gba.feature

```diff
@@ -0,0 +1,132 @@
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | gemeente van inschrijving (09.10) |
+      | 0530                              |
+      En de afnemer met indicatie '000008' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120                           | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 000008 |
+      | gemeenteCode | 0800   |
+      Als gba reisdocumenten wordt gezocht met de volgende parameters
+      | naam                    | waarde                         |
+      | type                    | RaadpleegMetReisdocumentnummer |
+      | reisdocumentnummer      | ID123NL45                      |
+      | gemeenteVanInschrijving | 0800                           |
+      | fields                  | reisdocumentnummer             |
+      Dan heeft de response 0 reisdocumenten
+
+    @fout-case
+    Scenario: Gebruik van de parameter gemeenteVanInschrijving is verplicht
+      Gegeven de afnemer met indicatie '000008' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120                           | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 000008 |
+      | gemeenteCode | 0800   |
+      Als gba reisdocumenten wordt gezocht met de volgende parameters
+      | naam                    | waarde                                                                     |
+      | type                    | ZoekMetBurgerservicenummer                                                 |
+      | burgerservicenummer     | 000000152                                                                  |
+      | fields                  | reisdocumentnummer,soort,datumEindeGeldigheid,inhoudingOfVermissing,houder |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                      |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1 |
+      | title    | Een of meerdere parameters zijn niet correct.               |
+      | status   | 400                                                         |
+      | detail   | De foutieve parameter(s) zijn: gemeenteVanInschrijving.     |
+      | code     | paramsValidation                                            |
+      | instance | /haalcentraal/api/reisdocumenten                            |
+      En heeft het object de volgende 'invalidParams' gegevens
+      | code     | name                    | reason                  |
+      | required | gemeenteVanInschrijving | Parameter is verplicht. |
```

Comment:
de afnemer die geen gemeente is komt niet bij de API terecht (want wordt door de API Gateway niet toegelaten). We hebben het vanmorgen gehad over het "vangnet" als men onverhoopt toch bij de API uit komt. Hiervoor geen scenario waarbij er geen gemeentecode in het token is opgenomen?

Reply:
probleem is dat de huidige opzet van de IDP in onze developomgeving dat niet ondersteunt. Die kunnen we wel puur voor documentatie opnemen, maar kan niet uitgevoerd worden in de testautomation